addons/mail/static/src/web/suggested_recipient.js

```diff
@@ -44,6 +44,8 @@ export class SuggestedRecipient extends Component {
                     default_name: this.props.recipient.name,
                     default_lang: this.props.recipient.lang,
                     force_email: true,
+                    ...(Object.fromEntries(Object.entries(
+                        this.props.recipient.customerInfo).map(([k, v]) => ['default_' + k, v]))),
```

Comment:
Note that the rest of the Discuss codebase is formatted according to the Framework JS prettier config, which you can install using the `reload.sh` script in `addons/web/tooling`

Reply:
Did not notice it was off, it stops working sometimes with multiple VS Code instances, thanks!